presto-geospatial-toolkit/src/test/java/com/facebook/presto/geospatial/TestGeometryUtils.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geospatial;
+
+import com.esri.core.geometry.ogc.OGCGeometry;
+import org.locationtech.jts.geom.Envelope;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestGeometryUtils
+{
+    private void assertJtsEnvelope(String wkt, Envelope expected)
+    {
+        OGCGeometry geometry = OGCGeometry.fromText(wkt);
+        Envelope calculated = GeometryUtils.getJtsEnvelope(geometry);
+        assertEquals(calculated, expected);
+    }
+
+    @Test
+    public void testGetJtsEnvelopeEmpty()
+    {
+        String wkt = "MULTIPOLYGON EMPTY";
+        Envelope expected = new Envelope();
+        assertJtsEnvelope(wkt, expected);
```

Comment:
- shorten to `assertJtsEnvelope("MULTIPOLYGON EMPTY", new Envelope());`
- ditto other tests